demisto_sdk/commands/common/hook_validations/incident_field.py

```diff
@@ -3,7 +3,8 @@
 """
 import re
 from enum import Enum, IntEnum
-
+from distutils.version import LooseVersion
+from demisto_sdk.commands.common.constants import FileType
```

Comment:
it fails on isort, meaning your imports are not in line. try:
```
import re
from distutils.version import LooseVersion
from enum import Enum, IntEnum

from demisto_sdk.commands.common.constants import FileType
from demisto_sdk.commands.common.errors import Errors
from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
    ContentEntityValidator
```
and read: https://www.python.org/dev/peps/pep-0008/#imports